example/assets/styles/style.css

```diff
@@ -7,8 +7,16 @@
   Required Styles
 \**************************/
 
-.modal[aria-hidden="true"] {
-  display: none;
+.modal {
```

Comment:
@kalpeshsingh Hey, why do we need this changes?

Reply:
This is some issue with `position: fixed` and `transform`. This workaround is require to work animation.